lib/rules/headers/copyright.js

```diff
@@ -42,6 +45,17 @@ exports.check = async function (sr, done) {
     if (copyright) {
         let allowBothLicense = false;
         const chartersData = await sr.getChartersData();
+
+        // Skip check if the document is only published by TAG and/or AB
+        const TagID = util.TAG.id;
+        const AbID = util.AB.id;
+        const deliverers = await sr.getDelivererIDs();
+        // groupIds: a list of ids without TAG or AB
+        const groupIds = deliverers.filter(
+            deliverer => [TagID, AbID].indexOf(deliverer) < 0
+        );
+        if (!groupIds.length) return done();
```

Comment:
The check is wrong and doesn't match the comment l.49.
If a document is a joint publication between the TAG (or AB) and a WG, then, we should look for the license associated with the WG.

Reply:
Let's take an example:
```
const groupIds = [TagID, AbID, WG_ID].filter(
    deliverer => [TagID, AbID].indexOf(deliverer) < 0
);
````

groupIds after `filter` would be 'WG_ID', which means L57 shouldn't execute.

---
> we should look for the license associated with the WG

The license is related to the charter. And the charter's data comes from `getChartersData()`, which will also skip TAG and AB, only return information regarding the `WG`

Reply:
Ok, I got confused bu the `indexOf`. Note that `includes` will be more accurate than `indexOf` because we are checking exact IDs.